herbie_core/admin/business_entity.py

```diff
@@ -17,7 +17,7 @@ def has_view_permission(self, request, obj=None):
         if super().has_view_permission(request, obj):
             return True
 
-        return request.user.has_perm('herbieapp.view_business_entities')
+        return request.user.has_perm('herbie.view_business_entities')
```

Comment:
must this also be renamed to herbie_core?

Reply:
:+1: good catch